### ISSUE TITLE
Set up issues

### ISSUE THREAD

**Issue by dgwyer** [boxyhq/saas-starter-kit#185]
Just trying out the project locally, following along with the README.md instructions.

I get as far as step 6 to set up the db schema: `npx prisma db push`

 I'm seeing the following error:

`Error: P1000: Authentication failed against database server at `localhost`, the provided database credentials for `admin` are not 
valid. Please make sure to provide valid database credentials for the database server at `localhost`.`

I'm using a Docker container for the db and that all seems OK so not sure why I'm getting this error. Any ideas?

See the screenshot for details of my `.env` file.

![image](https://github.com/boxyhq/saas-starter-kit/assets/1482075/8b6e1e6a-8251-4a5c-81c8-a4bf88413cd0)

![image](https://github.com/boxyhq/saas-starter-kit/assets/1482075/6ae1d3e0-eb37-43ed-8d49-d9cacbab95ad)

**Comment by devkiran** [comment]
I can see 2 database containers running in Docker. Can you make sure that you are using the correct one? I assume you're running Next.js app locally and not in Docker.

**Comment by kubeworkz** [comment]
You need the docker IP address for the connection. Usually 172.170.1. (not localhost)

**Comment by kubeworkz** [comment]
Can I see your db connection string in .env?

**Comment by dgwyer** [comment]
These are the two addresses, neither seem to work:

```
"Gateway": "172.19.0.1",
"IPAddress": "172.19.0.2",
```

Can anyone else confirm (on Windows specifically perhaps) the steps to set up via Docker and see if this is a general issue or just something my end.

**Comment by kubeworkz** [comment]
any Windows firewall blocking anything?

**Comment by devkiran** [comment]
@dgwyer did you able to run the app locally?

**Comment by Jmackles** [comment]
> Still the same issue unfortunately.

To piggyback on this, I had a similar issue, and I think I gotchu if you're still caring about this 3 months later lol. 

> any Windows firewall blocking anything? https://forums.docker.com/t/cant-connect-to-postgres-container-from-windows-10-but-can-from-wsl/128299/5

While the link provided earlier had the op's issue be framed as port related when they made their thread, I read through the thread and found the cause and resolved my issue (although I think someone should tell me if I did it the bad way).

The TL;DR of it is that if you have postgresql installed on your machine and have used localhost:5432 for it, it defaults to the system one I guess when you try to run `npx prisma db push`. I went into my `services` app on my windows machine and located the running postgresql services and just stopped them. I don't know if that was a stupid move in the long run but it got the job done. Anywho hope this is of some use either to you or future peeps.
